api/rs/slint/lib.rs

```diff
@@ -352,6 +354,7 @@ pub fn spawn_local<F: core::future::Future + 'static>(
 /// available for you to instantiate.
 ///
 /// Check the documentation of the `slint-build` crate for more information.
+#[doc = i_slint_core_macros::slint_doc_str!("Check the documentation of the [`slint-build`](slint:rust:slint_build/) crate for more information.")]
```

Comment:
(for next time: you can use the `#[i_slint_core_macros::slint_doc]` (without the str) so you don't need to put the thing in a string and can use normal comment)

Reply:
That's good to know :)